editor/debugger/editor_visual_profiler.cpp

```diff
@@ -647,10 +650,26 @@ void EditorVisualProfiler::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("enable_profiling", PropertyInfo(Variant::BOOL, "enable")));
 }
 
+void EditorVisualProfiler::_set_button_text() {
```

Comment:
It should be called `_update_button_text()`, because it's not a setter.

Reply:
changed for both "Profiler" and "Visaul Profiler". thanks for suggestion.